temporalio/bridge/client.py

```diff
@@ -0,0 +1,67 @@
+import os
+import socket
+from dataclasses import dataclass, field
+from typing import Mapping, Optional, Type, TypeVar
+
+import google.protobuf.message
+import temporal_sdk_bridge
+
+import temporalio.api.workflowservice.v1
+
+
+@dataclass
+class ClientTlsConfig:
```

Comment:
Because in Rust, these configurations are `Client` specific, not to be confused with worker configurations. These map to Rust types so I keep the name.

Reply:
The user facing `TLSConfig` is named as expected

Reply:
The user facing one is what matters